fuzzers/qemu/qemu_cmin/src/fuzzer.rs

```diff
@@ -223,8 +220,10 @@ pub fn fuzz() -> Result<(), Error> {
 
     let modules = tuple_list!(EdgeCoverageChildModule::default(),);
 
-    let emulator =
-        Emulator::new_with_qemu(qemu, modules, NopEmulatorExitHandler, NopCommandManager)?;
+    let emulator = EmulatorBuilder::empty()
```

Comment:
Should be `Emulator::builder()` (you can just create a function that forwards to this function)